test/CadlRanchProjects/azure/example/basic/tspconfig.yaml

```diff
@@ -0,0 +1,4 @@
+options:
+  "@azure-tools/typespec-csharp":
+    clear-output-folder: true
+    examples-dir: "{project-root}/../../../../../node_modules/@azure-tools/cadl-ranch-specs/http/azure/example/basic/examples"
```

Comment:
I thought TCGC has a default value of `./examples`, do we have to configure this to make it work?

Reply:
Because this project is intended to test loading JSON examples and generating corresponding sample code, and the corresponding Example folder is not placed in this project but in Cadl Ranch, I have tried many different file path formats to read the JSON files in the Example folder. However, only the current format can successfully achieve the goal. I am not sure if there are other more appropriate path expressions, but as far as I know, only this format can successfully run at the moment.